tests/transactions/builder/test_multi_payment.py

```diff
@@ -1,8 +1,23 @@
-import pytest
+from crypto.constants import TRANSACTION_MULTI_PAYMENT, TRANSACTION_TYPE_GROUP
+from crypto.transactions.builder.multi_payment import MultiPayment
 
 
-@pytest.mark.skip(reason='not implemented')
 def test_multi_payment_transaction():
     """Test if multi payment transaction gets built
     """
-    pass
+    transaction = MultiPayment()
+    transaction.set_type_group(TRANSACTION_TYPE_GROUP.CORE)
+    transaction.set_nonce(1)
+    transaction.add_payment(1, 'AHXtmB84sTZ9Zd35h9Y1vfFvPE2Xzqj8ri')
```

Comment:
multipayment minimum is two payments, so I rather see two in the test too. This will also make the test more future-resilient if we decide to have the builder do more explicit checks and for example disallow a single payment to be built